### PR TITLE
Filename convention and BasisPoint definition

### PR DESCRIPTION
**Pull request by muichkine**
- A couple of file renaming to follow the repo convention
- Centralized the definition of a basis point to Constants.BasisPoint

### CODE REVIEW

File: src/QLNet/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+namespace QLNet
+{
+    public struct Constants
```
Comment (amaggiulli) [inline src/QLNet/Constants.cs]: Can you insert into existing : public struct Const in Types.cs ?

Comment (muichkine) [reply src/QLNet/Constants.cs]: Sure thing.

Comment (muichkine) [reply src/QLNet/Constants.cs]: Done.

**Comment by igitur** [comment]
This PR introduced unnecessary whitespace changes. Please remember to run `.\format_code.bat` before committing.

**Comment by muichkine** [comment]
ok @igitur, will do in the future. thanks.

**Pull request merged**